uitest/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+    // This JSON file is the default app settings file. Other
+    // appsettings.*.json files can overwrite its values.
+    "WindowsApplicationDriverUrl": "http://127.0.0.1:4723",
+    "PackageFamilyName": "Microsoft.Windows.DevHome.Dev_8wekyb3d8bbwe",
+    "Widget": {
+        "IdPrefix": "Microsoft.Windows.DevHome.Dev_8wekyb3d8bbwe",
+        "Provider": "CoreWidgetProvider"
```

Comment:
I am not very familiar with the widget provider, so I am not sure if there's value in adding the `CoreWidgetProvider` here (let me know if you have preference). This and the `IdPrefix` are used to determine which NavigationItem to click in the dialog page, which allows UI testing Dev Home (Canary) with Dev Home (Dev) widgets.

More details: AddWidgetDialog.cs

CC: @krschau @ujjwalchadha